### ISSUE TITLE
Rule ignoring «contains: screenshot» 

### ISSUE THREAD

**Issue by iburunat** [tfeldmann/organize#176]
This rule ignores any *screenshot* files in the pertinent folder. 
Maybe somebody can see where the error is?
I need the rule to target any file whose filename contains `screenshot`.

```yaml
  - name: "move screenshots to temp folder"
    locations: '~/OneDrive/Pictures/Camera Roll/'
    subfolders: false
    enabled: true
    filters:
      - extension:
        - png
        - jpg
        - jpeg
      - name:
          contains: screenshot
          case_sensitive: false
    actions:
      - move:
          dest: "~/Pictures/album/TEMP/"
          on_conflict: rename_new
          rename_template: "{name}_{counter}{extension}"
```

**Comment by tfeldmann** [comment]
Just tried it and cannot reproduce. Are you sure there are screenshot files in the given location?

**Comment by iburunat** [comment]
That's strange.
I have the following files in the origin folder:

![image](https://user-images.githubusercontent.com/26675866/153221142-9e373105-8b29-4bd4-b7f1-615f1af8ad44.png)

And when I simulate or run organice, it spits `Nothing to do`.

**Comment by iburunat** [comment]
I really don't wish to enable subfolders, because this would mean applying the rule on other screenshots located in subfolders (which I don't need to process).

**Comment by tfeldmann** [comment]
This is very strange, works perfectly here. Really cannot explain it so far.

```sh
❯ tree ~/Desktop/issue-\#176 
/Users/tf/Desktop/issue-#176
├── Screenshot1.jpg
├── abcScreenshotabc.jpg
├── abcscreenshot.jpg
└── screenshot.jpg

0 directories, 4 files

❯ organize sim .configs/28-issue-176.yaml         
organize 2.0.8
Config: ".configs/28-issue-176.yaml"

╭─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ SIMULATION                                                                                                                                                  │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯

⚙ move screenshots to temp folder ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
/Users/tf/Desktop/issue-#176
  abcScreenshotabc.jpg 
    - (move) Move to /Users/tf/Pictures/album/TEMP/abcScreenshotabc.jpg
  screenshot.jpg 
    - (move) Move to /Users/tf/Pictures/album/TEMP/screenshot.jpg
  Screenshot1.jpg 
    - (move) Move to /Users/tf/Pictures/album/TEMP/Screenshot1.jpg
  abcscreenshot.jpg 
    - (move) Move to /Users/tf/Pictures/album/TEMP/abcscreenshot.jpg

╭─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ SIMULATION                                                                                                                                                  │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯

success 4 / fail 0

❯ cat .configs/28-issue-176.yaml 
rules:
  - name: "move screenshots to temp folder"
    locations: "~/Desktop/issue-#176"
    subfolders: false
    enabled: true
    filters:
      - extension:
          - png
          - jpg
          - jpeg
      - name:
          contains: screenshot
          case_sensitive: false
    actions:
      - move:
          dest: "~/Pictures/album/TEMP/"
          on_conflict: rename_new
          rename_template: "{name}_{counter}{extension}"
```

Which OS + version are you on?

**Comment by iburunat** [comment]
Now I see the problem. This rule is conflicting with another one (which comes later in the YAML, which is strange, because I thought the processing was sequential). When I comment it out, it works fine. Otherwise, it returns `nothing to do`. Will inspect it later and come back with some insights :D

I will

**Comment by iburunat** [comment]
I forgot to answer the OS version earlier:  `MacOS Catalina 10.15.7`
Found the line that is interfering but can't explain why:

```yaml
rules:
  - name: "/CAMERAROLL > TEMP {SCREENSHOTS}"
    locations: '~/OneDrive/Pictures/Camera Roll/'
    subfolders: false
    enabled: true
    filters:
      - extension:
        - png
        - jpg
        - jpeg
      - name:
          contains: screenshot
          case_sensitive: false
    actions:
      - move:
          dest: "~/Pictures/album/TEMP/"
          on_conflict: rename_new
          rename_template: "{name}_{counter}{extension}"

# ----------------------------------------------------------------------------------|
  - name: "/CAMERAROLL > TEMP {IMAGES}"
    locations:
     - path: '~/OneDrive/Pictures/Camera Roll'
       exclude_files: ['*Screenshot*.*', '*screenshot*.*']  # <---- if commented, it works.
    subfolders: false
    enabled: true
    filters:
      - extension:
        - jpg
        - png
        - jpeg
    actions:
     - move:
         dest: "~/Pictures/album/TEMP/"
         on_conflict: rename_new
         rename_template: "{name}_{counter}{extension}"
```

If the line `exclude_files: ['*Screenshot*.*', '*screenshot*.*'] ` is commented in the second rule, the first rule works. How can a distinct rule interfere like this as if if was ignoring *screenshot* for all rules?

**Comment by tfeldmann** [comment]
That's a bug! And a strange one. I'll look into this 👌👌

**Comment by tfeldmann** [comment]
Should be fixed in v2.0.9! Thanks for reporting!

**Comment by iburunat** [comment]
Fantastic work, thanks. This issue is fixed. I have another one for which I will open a new issue. 
Also congrats on the new look for the output, very pleasant to look at and visually parse.
